### PR TITLE
Uploaded Coordination implementation

### PR DESCRIPTION
**Pull request by babbo65**
Code utilities Storage and PrimeSummation which implement the API's, should be all good but lmk if there are any errors with names or instances

### CODE REVIEW

File: src/Coordination.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import java.util.ArrayList;
+
+class Coordination {
+    //Global variables
+    private Computation engine;
+    private List<Integer> in;
+    private List<Integer> out;
+    private ComputeEngineAPI computAPI;
```
Comment (colinm0125) [inline src/Coordination.java]: computeAPI is spelled wrong here

File: src/Coordination.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    private DataToFromComputeAPI dataAPI;
+    private int key;
+
+    //Constructors
+    public Coordination(ComputeEngineAPI computeAPI) {
+        this.computAPI=computeAPI;
+    }
+    public Coordination() {
+        computeAPI = new ComputeEngineAPI();
```
Comment (colinm0125) [inline src/Coordination.java:17]: I see you used "computAPI" here, wanted to clarify if there is a spelling error or if this was intended to be named as such. If so, consider renaming for clarity

Comment (colinm0125) [reply src/Coordination.java:17]: On line 14 (oops) ^^

**Review by colinm0125 — COMMENTED** [review]
Besides potential spelling error/unclear variable naming looks good. The logic all seems to follow a good path.

File: src/Coordination.java (hunk 1 above)
Comment (babbo65) [reply src/Coordination.java]: Good catch thank you!

File: src/Coordination.java (hunk 2 above)
Comment (babbo65) [reply src/Coordination.java:17]: I see the confusion, I fixed both the error to be computeAPI, thank you! Should be good now if you want to re-review

**Review by colinm0125 — APPROVED** [review]
Awesome, perfect now

**Pull request merged**